src/main/java/teammates/ui/controller/InstructorCourseInstructorAddAction.java

```diff
@@ -98,9 +121,14 @@ private InstructorAttributes updateBasicInstructorAttributes(String courseId, St
         
         return instructorToAdd;
     }
-
-    private void updateInstructorCourseLevelPrivileges(
-            InstructorAttributes instructorToAdd) {
+    
+    /**
+     * Updates course level privileges for the instructor by retrieving request parameters.
+     * 
+     * @param instructorToAdd Instructor that will be added.
+     *                            This will be modified within the method.
+     */
+    private void updateInstructorCourseLevelPrivileges(InstructorAttributes instructorToAdd) {
```

Comment:
`instructorToAdd` or `instructorToUpdate`?
(this will bust some of the line length limit, but correctness takes precedence over anything else)